lms/djangoapps/teams/csv.py

```diff
@@ -226,15 +241,18 @@ def load_course_team_memberships(self):
     def load_course_teams(self):
         """
         Caches existing course teams by (team_name, topic_id)
+        and existing membership counts by (topic_id, team_name)
         """
         for team in CourseTeam.objects.filter(course_id=self.course.id):
             self.existing_course_teams[(team.name, team.topic_id)] = team
+            self.user_count_by_team[(team.topic_id, team.name)] = team.users.count()
```

Comment:
Could you add an assertNumQueries to a test that hits this? I think you may want to prefetch_related for this? Not positive

Reply:
Good catch 👍

Reply:
so that does help here? nice